wix-bookings-v2/wix-bookings-v2/SessionsView/guides/1. Introduction.md

```diff
@@ -10,3 +10,8 @@ The sessions view consists with only single sessions or expanded, recurring sess
 
 The sessions view has an end date, which is set to be at least a year into the future.  
 Periodically, the view is automatically extended, to preserve its future, 1 year, duration.
+
+## Limitations
```

Comment:
You know the Intro guidelines - please review them. 

1. You are missing the Terminology heading with the link to our main Terminology page. 
2. You are missing the "Before you begin" section which is what we use instead of "Limitations" 
3. "Session Participants" should not be a heading - it should be a bullet.   Like this: 

- **Session Participants**: Sessions include at most 5 participants. To fetch the entire list of participants, use the [Bookings XXX endpoint](yyy).